### ISSUE TITLE
Inlays discolour UI windows behind them when using GShade

### ISSUE THREAD

**Issue by Kanarisky** [ackwell/BrowserHost#15]
**Describe the bug here:**
Browser Host inlays seem to be doubly affected by GShade. This primarily affects game UI windows. However it's also visible on the Duty list UI and is plainly visible on the character selection screen.

**Steps to reproduce the bug:**
1. Install GShade and use a preset that noticeably impacts graphics
2. Install XIVLauncher + Browser Host. 
3. Setup a Browser Host inlay that should be transparent (such as a Cactbot Raidboss Timeline/Alerts URL)
4. Drag a game UI window (such as duty finder or inventory) underneath where you've placed your inlay.
5. Visit character selection screen.

**Expected behavior**
Nothing out of the ordinary. Inlay is transparent. No discolouration or graphical oddities on UI elements or the character selection screen.

**Actual behavior**
The area of the UI covered by the transparent inlay has been affected by GShade a 2nd time. Because of this, it is darker (most GShade presets darken/increase contrast/saturation) and looks very strange. Areas of character selection screen have dark rectangles.

**Screenshots**
Example 1 (Cactbot timeline-only inlay, hidden)
![Example 1 (inlay hidden)](https://user-images.githubusercontent.com/68185248/125381846-fb636100-e3d7-11eb-84f7-89455c0eaf4c.png)

Example 1 (Cactbot timeline-only inlay, transparent but not hidden)
![Example 1 (inlay transparent but not hidden)](https://user-images.githubusercontent.com/68185248/125381919-1e8e1080-e3d8-11eb-834a-99941726faba.png)

Example 2 (Cactbot alerts-only inlay, hidden)
![Example 2 (inlay hidden)](https://user-images.githubusercontent.com/68185248/125381971-31084a00-e3d8-11eb-8db1-56e756489b19.png)

Example 2 (Cactbot alerts-only inlay, transparent but not hidden)
![Example 2 (inlay transparent but not hidden)](https://user-images.githubusercontent.com/68185248/125382075-58f7ad80-e3d8-11eb-97e5-5cf106d262a8.png)

Example 3 (Char select, GShade OFF, both Cactbot inlays transparent but not hidden)
![Example 3 (Char select, GShade off)](https://user-images.githubusercontent.com/68185248/125382118-6ca31400-e3d8-11eb-9f65-b9c220605384.png)

Example 3 (Char select, GShade ON, both Cactbot inlays transparent but not hidden)
![Example 3 (Char select, GShade on - see where the Browser Host inlays are)](https://user-images.githubusercontent.com/68185248/125382151-7cbaf380-e3d8-11eb-8144-6c7362ae52c2.png)

**Other information:**
Operating System: Windows 10
XIVLauncher version: 6.0.4
Launching as admin?: Nope
Using Steam?: Yep
Addons/Plugins enabled: 
![These](https://user-images.githubusercontent.com/68185248/125382278-bab81780-e3d8-11eb-89e4-ffc9c2c9ff75.png)
 

**Additional information**
 GShade preset: SerenayaRealismGameplay
 
If I toggle GShade off, the problem does not occur. It seems to be an interaction between GShade and Browser Host. 

Setting Browser Host to the experimental render mode (BitmapBuffer) under advanced settings does nothing to alleviate the issue.

Additionally, this problem does not occur with transparent RemindMe windows, so there must be a difference between how RemindMe and Browser Host render their stuff.

**Comment by DAOWAce** [comment]
Can confirm here.

Have to use this plugin to workaround ACT disabling NVIDIA's Gsync when any overlay is drawn.. so now all the overlays get affected by GShade, and subsequently affect the game UI even through KeepFX.

The problems with compatibility between programs just keep adding up..

**Comment by willho01** [comment]
Did you ever find a workaround for this? Having the same issue.

**Comment by Kanarisky** [comment]
Nope! I think(?) Attick said that because it's to do with GShade, they won't be investing time into trying to fix it. Seems like it's out of their control.

I ended up uninstalling GShade, sorry.

**Comment by willho01** [comment]
Fair enough. Well for now I guess I'll just use ACT to display the overlays rather than browser host. Shame.
